Gemfile

```diff
@@ -38,6 +38,8 @@ group :development do
 
   # passenger tasks in capistrano
   gem 'capistrano-passenger'
+  
+  gem 'meta_request'
```

Comment:
Oh shit, didn't mean to leave this in, you guys cool with it? It's a chrome extension to debug [RailsPanel](https://github.com/dejan/rails_panel).